cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -54,6 +54,10 @@
   {% set bind_address = "--bind-address=" + grains.publicAddressOverride -%}
 {% endif -%}
 
+{% set storage_backend = "" -%}
+{% if pillar['storage_backend'] is defined -%}
+  {% set storage_backend = "--storage-backend=" + pillar['storage_backend'] -%}
```

Comment:
That is my understanding - when I run it in my local project with etcd3 storage backend, it seemed to work (and there weren't any objects under etcd:4001/v2/keys/, which suggests that it was really using v3 API and storage.